libvast/src/system/importer.cpp

```diff
@@ -226,9 +232,16 @@ void importer_state::send_report() {
   using namespace std::string_literals;
   auto elapsed = std::chrono::duration_cast<duration>(now - last_report);
   auto node_throughput = measurement{elapsed, measurement_.events};
+  auto num_schemas_seen = schema_counters.size();
+  std::vector<performance_sample> samples = {};
+  samples.reserve(num_schemas_seen + 2);
+  samples.emplace_back("importer"s, measurement_);
+  samples.emplace_back("node_throughput"s, node_throughput);
+  for (const auto& [name, count] : schema_counters)
+    samples.emplace_back(name, measurement{elapsed, count});
```

Comment:
I think we should probably prefix these keys with `"schema"` or something similar.

After the key names have been decided the docs need to be updated.

Reply:
> For the new metrics my preference would be just `throughput`, with the schema name as an additional metadata field so we can choose to either render accumulated values or to render the values individually.

It looks like we need to agree on the general structure of the metrics hierarchy first. Proposal: let's take a step back and agree on an intuitive scheme, then evaluate concrete suggestions.

Reply:
> I'm against putting the schema name in the metric key itself, as that makes it rather hard to use the data from a dashboarding perspective.

I didn't consider this, but I guess you're right. You're run-of-the-mill dashboard won't be able to prefix-match on metrics keys.

I would still like to use the `ingest` identifier for the top-level. The key would then be `ingest.schemas` with each value tagged by the individual schema names. We can also add a value with the `total` tag that holds the same value as the current `node_throughput`.

Reply:
Stacking the lines makes it really hard to see how the rates relative to each other and jitters propagate upwards. Maybe we should just keep `node_throughput` as a separate key for those who want the total counter.

Reply:
Having the total in a separate key sounds good to me.